.github/workflows/cqrs-publish.yml

```diff
@@ -24,7 +24,7 @@ jobs:
       - name: Setup Dart
         uses: dart-lang/setup-dart@v1
         with:
-          sdk: 2.18.6
+          sdk: 3.0.1
```

Comment:
Here you don't use quotes, but in other places you do.
Is there any difference?

Reply:
I am no yaml expert, but I think both produce strings? I just replaced whatever was there already. We can change every occurrence to be consistent

Reply:
I prefer to not use quotes in YAML wherever possible, but when it comes to semver versions such as this, I like to put single quotes around it. I recall there are some wild problems with sth like `0.2.3` being treated as octal (I honestly don't remember).

Reply:
Would you like me to change all occurrences of version strings to be quoted?

Reply:
friendly ping @bartekpacia

Reply:
Sorry for the lag.

I'd prefer to not use quotes at all, and only add them if something breaks.